WalletWasabi.Fluent/ViewModels/Wallets/Send/TransactionPreviewViewModel.cs

```diff
@@ -411,10 +414,15 @@ protected override void OnNavigatedTo(bool isInHistory, CompositeDisposable disp
 	{
 		base.OnNavigatedTo(isInHistory, disposables);
 
+		if (_coinJoinManager is { } coinJoinManager)
+		{
+			coinJoinManager.WalletEnteredTxPreview(_wallet);
```

Comment:
Why not here? https://github.com/molnard/WalletWasabi/blob/master/WalletWasabi.Fluent/ViewModels/Wallets/Send/SendViewModel.cs#L130

Reply:
https://github.com/zkSNACKs/WalletWasabi/pull/10500#discussion_r1177441624

Reply:
https://github.com/zkSNACKs/WalletWasabi/pull/10500#discussion_r1180464727